src/main/external-resources/renderers/LG-LS5700.conf

```diff
@@ -41,9 +40,6 @@ Supported = f:mp3            n:2             m:audio/mpeg
 Supported = f:wav   a:lpcm   n:6   s:48000   m:audio/wav
 Supported = f:wma            n:2   s:48000   m:audio/x-ms-wma
 
-# Supported image formats:
-Supported = f:jpg   m:image/jpeg
```

Comment:
Yes, but again it's done according to DLNA. Images are offered in JPEG, PNG and GIF if the source is GIF. Those are the only three image formats supported by DLNA. All images will be offered in both JPEG and PNG, and will be "automatically transcoded" to the format the renderer is requesting. The ```CI``` flag is used correctly to tell the renderer which offers are transcoded and which are original. Renderers are supposed to prefer the non-transcoded content if they support it.

It's been a goal for me to move away from renderer configurations where possible, because they only work well where the renderer configuration is optimal. This will work as well for any renderer, even those without a renderer profile.